Raven.Database/Raft/Controllers/ClusterAdminController.cs

```diff
@@ -143,14 +143,34 @@ private bool ConfigurationsAreEqualExceptReplicationCheck(ClusterConfiguration c
         [RavenRoute("admin/cluster/initialize-new-cluster/{*id}")]
         public HttpResponseMessage InitializeNewCluster(string id)
         {
+            var oldClusterManager = SetClusterManagerToNullAndGetOldValue();
+            if (id != null)
+            {
+                oldClusterManager.CleanupAllClusteringData(SystemDatabase);
```

Comment:
this was a surprise to me too but cluster manager is always there even when not in cluster.
You're right that if two concurrent calls to this endpoint will occur this might throw NRE on the other but many other endpoints will also throw NRE it is expected (those calls were actually the race condition causing all the madness)